lms/static/sass/views/_login-register.scss

```diff
@@ -10,13 +10,34 @@
     background: $white;
     min-height: 100%;
     width: 100%;
+    padding-left: ($baseline/2);
+    padding-right: ($baseline/2);
     $third-party-button-height: ($baseline*1.75);
 
     h2 {
-        @extend %t-title5;
+        @extend %t-title4;
+        margin-top: $baseline;
+        letter-spacing: normal;
+        font-family: $sans-serif;
+        color: $uxpl-blue-hover-active;
+    }
+
+    h3 {
+        @extend %t-title6;
+        @extend %t-light;
         margin: 0;
         letter-spacing: normal;
         font-family: $sans-serif;
+        color: $gray-d4;
+    }
+
+    h3.fields-required {
+        font-size: 14px;
+        color: #414141;
```

Comment:
This should be a variable from http://ux.edx.org/design_elements/colors/ not a hex code.